xml/XSLT/panelfile-4-19-2.xsl

```diff
@@ -507,7 +540,7 @@ Logic delay: <xsl:value-of select="logicDelay"/> (ms)<br/>
         <xsl:param name="name"/>
     <!-- index through individual conditional elements, looking for match -->
     <xsl:for-each select="/layout-config/conditionals/conditional">
-    <!-- BUG 2022-02-15: if logix system name has embedded lower case letters, next statement always fails
+    <!-- BUG 2022-02-15: if logix system name has embedded lower case letters, next statement always fails 
```

Comment:
@danielb987 Do you know if Logix (not LogixNG) finds things in a case-insensitive way?  if so, that sounds like the underlying problem, but it would be hard to fix at this point.

Reply:
Lower case letters in internal system names are valid.  Logix does not have an issue.  The comment and original translate line are not needed.

Reply:
Done.